src/core/lombok/core/handlers/HandlerUtil.java

```diff
@@ -315,6 +315,18 @@ public static int primeForNull() {
 			"com.fasterxml.jackson.annotation.JsonProperty",
 			"com.fasterxml.jackson.annotation.JsonSetter",
 		}));
+		JACKSON_COPY_TO_BUILDER_ANNOTATIONS = Collections.unmodifiableList(Arrays.asList(new String[] {
```

Comment:
Looks like a reasonable list: the only one I was thinking might make sense to add would be `JsonAutoDetect`?

Reply:
So I suppose Jackson will just evaluate `setterVisibility` in the builder case, right?

Reply:
Jackson will actually do full visibility evaluation for all accessors since sometimes visible getter is required to pull in non-visible setter. But for most cases I assume `setterVisibility` / `fieldVisibility` is what really matters.

Reply:
I created PR #2475 for this.

Reply:
What about com.fasterxml.jackson.annotation.JsonAnyGetter and com.fasterxml.jackson.annotation.JsonAnySetter?

Reply:
`@JsonAnyGetter` is only relevant for serialization, so there is no use in copying it to a builder.

A `@JsonAnySetter` method is called once for every attribute that is not directly mapped to a field. Considering lombok, it will work only for the singular method of a `@Singular Map<String, Object>` field, and this is exactly where the current edge release will copy it to.

Is there another use-case I don't see?